docs/output.md

```diff
@@ -253,8 +253,9 @@ Sequencing depth per contig and sample is generated by `jgi_summarize_bam_contig
 <summary>Output files</summary>
 
```

Comment:
Ok, so the pipeline output is perfect now, just the output description is now redundant for  `GenomeBinning/MetaBAT2/discarded/`, in lines 258 **and** 267-269. Also, `GenomeBinning/MetaBAT2/unbinned/` is refered to in lines 257 **and** 270.

Line 262 reads `All the files and contigs in this folder will be assessed by QUAST and BUSCO.` and therefore folder `discarded` should be not included here, i.e. line 258 should be deleted.

The files `unbinned.pooled,unbinned.remaining` are now in folder discarded, so line 270 should be deleted.

I attempt to make suggestions below to solve that.
However, I could not make a suggestion for Line 262, which includes now 2 folders and therefore probably should be `All the files and contigs in those folders will be assessed by QUAST and BUSCO.`

Reply:
Updated! Hope everything shuld be clear now :)